src/js/data/CosmosClient.ts

```diff
@@ -0,0 +1,28 @@
+import { request, RequestResponse } from "@dcos/http-service";
```

Comment:
Let's wrap  the client as an npm package and move it to `packages/cosmos-client` 💪 with all package.json etc.

Reply:
is something we want to do for all new clients? ref: #3579